security_monkey/alerters/custom_alerter.py

```diff
@@ -14,32 +14,59 @@
 """
 .. module: security_monkey.alerters.custom_alerter
     :platform: Unix
-
-.. version:: $$VERSION$$
-.. moduleauthor:: Bridgewater OSS <opensource@bwater.com>
-
-
+.. version:: SlackBot Integration Alerter w/ Security_Monkey
+.. moduleauthor:: Drew Gallis <drew.gallis@willowtreeapps.com>
 """
 from security_monkey import app
+from slack_post import postMessage
+import os
 
 alerter_registry = []
 
-
 class AlerterType(type):
-
-    def __init__(cls, name, bases, attrs):
+    print(os.environ) #remove if you don't need to check your env for your slack api token
+    def __init__(self, cls, name, bases, attrs):
         if getattr(cls, "report_auditor_changes", None) and getattr(cls, "report_watcher_changes", None):
             app.logger.debug("Registering alerter %s", cls.__name__)
             alerter_registry.append(cls)
 
-
+#UNCOMMENT ALL POSTMESSAGE FUNCTIONS WHEN YOUR SLACKCLIENT IS SETUP
```

Comment:
I think the code should be set up such that an env variable or check is in place to only send messages when the slack client is fully operational.

Users should not have to make any modifications or changes to this code.